src/App.ts

```diff
@@ -400,10 +406,10 @@ export default class App {
     // Factory for say() utility
     const createSay = (channelId: string): SayFn => {
       const token = context.botToken !== undefined ? context.botToken : context.userToken;
-      return (message: Parameters<SayFn>[0]) => {
+      return (message: Parameters<SayFn>[0]): Promise<void | WebAPICallResult> => {
         const postMessageArguments: ChatPostMessageArguments = (typeof message === 'string') ?
           { token, text: message, channel: channelId } : { ...message, token, channel: channelId };
-        this.client.chat.postMessage(postMessageArguments)
+        return this.client.chat.postMessage(postMessageArguments)
```

Comment:
I thought this made more sense resolving the result of this call rather than `void`. 

Enables folks who want to the ability to get things like the `ts` off the result for later updating.